.github/workflows/tests.yml

```diff
@@ -4,16 +4,17 @@ name: Test suite
 
 on:
   pull_request:
+    types: [ opened, synchronize, reopened, ready_for_review ]
     branches: [ master ]
 
 jobs:
-  lint-unittests-docs:
-    name: Lint, Unit tests and Docs
+  lint-unittests-docs-build:
+    name: Lint, Unit tests, Docs and Build
```

Comment:
These two lines state that docs and python packages are built, but they have been removed below.